react/modules/mediaAsList.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+
+import Media from '../Media'
+import type {
+  ImageModeProps,
+  VideoModeProps,
+  ImageListSchemaElement,
+  MediaArray,
+  VideoListSchemaElement,
+  MediaListSchemaElement,
+  MediaProps,
+} from '../MediaTypes'
+
+export const getMediaAsJSXList = (
+  mediaList: MediaArray,
+  isMobile: boolean,
+  maxHeight?: string | number
+) => {
+  return mediaList.map((props: MediaProps | MediaListSchemaElement, idx) => {
```

Comment:
I don't have too much context, but use the index as a component key isn't too good.

Reply:
We chose to maintain that way to ensure it's backwards compatible with `list-context.media-list`.